.github/workflows/snyk.yml

```diff
@@ -10,8 +10,11 @@ jobs:
   security:
     uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@main
     with:
-      DEBUG: true
+      DEBUG: false
       ORG: guardian
       SKIP_NODE: false
+      SKIP_PYTHON: false
+      PIP_REQUIREMENTS_FILES: quarantine-status/lambda/requirements.txt
+      PYTHON_VERSION: 3.7.13
```

Comment:
the python version used in the lambda is 3.6 but ubuntu 22.04 only supports 3.7.13 and above, so this is as close as we could get it. Might be worth bumping the runtime at some point, but this is an extremely minor concern